deployment/playbooks/remove-volttron.yml

```diff
@@ -0,0 +1,32 @@
+- import_playbook: includes/ensure_host_keys.yml
+
+- hosts: all
+  tasks:
+    - include_tasks: includes/common.yml
+
+    - name: Configure VOLTTRON Instances
+      volttron_instance_config:
+        config_file: "{{ host_config }}"
+        state: "STOPPED"
+        phase: "UNINSTALL"
+      register: instance_state
+      ignore_errors: yes
+
+    - name: Remove vollttron directory, files and other volttron artifacts
+      file:
+        state: absent
+        path: "{{ item }}"
+      with_items:
+        - "{{ volttron_home }}"
+        - "{{ volttron_root }}"
+        - "{{ host_config }}"
+        - "{{ my_ansible_vitualenv }}"
+        - "{{ ansible_env.HOME }}/rabbitmq_server"
+        - "{{ ansible_env.HOME }}/.volttron_rmq_home"
+        - "{{ ansible_env.HOME }}/.volttron_instances"
+        - "{{ ansible_env.HOME }}/ansible_logging.log"
+        - "{{ ansible_env.HOME }}/logfile.log"
+        - "{{ ansible_env.HOME }}/configs"
+        - "{{ ansible_env.HOME }}/volttron.tar.gz"
+
```

Comment:
Ben suggests there is more efficient to handle this. He is taking it as action item for later.